src/components/about/index.css

```diff
@@ -0,0 +1,29 @@
+.whoAreWe {
+  color: #161b50;
+  font-weight: bold;
+  font-size: 4rem;
+  margin-bottom: 40px;
+  text-transform: uppercase;
+}
+
+.aboutParagraphs {
+  font-size: 1.35rem;
+  margin-bottom: 109px;
+}
+
+p {
```

Comment:
i believe these rules are global, so by doing this you might be affecting all your teammates p. i'm not sure, but either way, it's not a good practice to apply rules too broadly to html tags like this, so use a class or scope it